tests/signing/testdata/signing/TestSigningPayload/deep_model_folder_shard

```diff
@@ -0,0 +1,42 @@
+{
+  "_type": "https://in-toto.io/Statement/v1",
+  "subject": [
+    {
+      "name": "deep_model",
+      "digest": {
+        "sha256": "18b5a45fe7983f7194e8ffd96c80f5f0ec53191bf4a32b6aff293f043e816d7a"
+      }
+    }
+  ],
+  "predicateType": "https://model_signing/signature/v1.0",
+  "predicate": {
+    "serialization": {
+      "method": "shards",
+      "hash_type": "sha256-sharded-1000000000",
+      "allow_symlinks": true,
+      "shard_size": 1000000000.0
```

Comment:
shard size has to be an integer right? is this just a JSON quirk?

Reply:
I think so, though when I tested from python interpreter with `json.dumps` it used int. I'll debug when I get to more testing e2e (which your PR helps with)